### PR TITLE
API-6770 - Replace Joda Time with Java Time

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: app/uk/gov/hmrc/gatekeeper/models/Application.scala [hunk 1]
```diff
@@ -432,7 +425,7 @@ object CollaboratorRole extends Enumeration {
   implicit val format = Json.formatEnum(CollaboratorRole)
 }
 
-case class ApplicationState(name: State = State.TESTING, requestedByEmailAddress: Option[String] = None, verificationCode: Option[String] = None, updatedOn: DateTime = DateTime.now()) {
+case class ApplicationState(name: State = State.TESTING, requestedByEmailAddress: Option[String] = None, verificationCode: Option[String] = None, updatedOn: LocalDateTime = LocalDateTime.now()) {
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/gatekeeper/models/Application.scala:428]: Do these not need to be Zoned?
I am not certain myself but I think it's worth asking.

Comment (peteslater-ee) [reply app/uk/gov/hmrc/gatekeeper/models/Application.scala:428]: Like LocalDateTime.now(ZoneOffset.UTC)

Comment (AndySpaven) [reply app/uk/gov/hmrc/gatekeeper/models/Application.scala:428]: I really hope we are not mixing zones in now...

**Review by mattclark-zerogravit — APPROVED** [review]
👍

**Pull request merged**